src/auth/auth.controller.ts

```diff
@@ -80,19 +82,28 @@ export class AuthController {
 
     let message = '';
     if (authDetails.status === SgidAuthStatus.AUTHENTICATED_PUBLIC_OFFICER) {
-      const verifiedMessage = authDetails.poDetails.map((object) => {
-        return `You are verified with the following details:\n
-Agency: ${object.agency_name}
-Department: ${object.department_name}
-Title: ${object.employment_title}`;
+      const verifiedMessage = [`You are verified with the following details:`];
+      for (const poDetail of authDetails.poDetails) {
+        verifiedMessage.push(
+          `Agency: ${poDetail.agency_name} \n Department: ${poDetail.department_name} \n Title: ${poDetail.employment_title}`,
+        );
+      }
+
+      await this.databaseService.store.set(userId, {
+        name: authDetails.name,
+        verified_date: new Date(),
+        emails: authDetails.poDetails.map((poDetail) => poDetail.work_email),
```

Comment:
Could we store the raw poDetails as is, and leave it to the user to decide what to include? This also depends on whether there is information we wish to persist at all